apps/server/src/modules/h5p-editor/service/h5p-editor.service.ts

```diff
@@ -13,6 +13,7 @@ export const H5PEditorService = {
 		const config: H5PConfig = new H5PConfig(undefined, {
 			baseUrl: '/api/v3/h5p-editor',
 			contentUserStateSaveInterval: false,
+			contentHubEnabled: false,
```

Comment:
```suggestion
		const config: H5PConfig = new H5PConfig(undefined, {
			baseUrl: '/api/v3/h5p-editor',
			contentUserStateSaveInterval: false,
			contentHubEnabled: false,
```

Can H5PConfig exist once for player and editor?

Reply:
Moved config to a shared file